R/data.R

```diff
@@ -0,0 +1,89 @@
+#' Dummy Single-arm Trial Data for Unanchored MAIC
+#'
+#' These datasets contains adsl_sat, adtte_sat, adrs_sat,
+#' agd_sat (AgD of effect modifiers), pseudo_ipd_sat (AgD, tte data)
+#' @format **adsl_sat**: a data frame with xx rows and x columns:
+#'   \describe{
+#'     \item{id}{The ID of the patients.}
+#'   }
```

Comment:
We can have all the anchored data sets documented in one page. What do you think? Same for unanchored
@hoppanda